gamestonk_terminal/common/quantitative_analysis/qa_view.py

```diff
@@ -947,7 +947,13 @@ def display_line(
 
 
 def display_var(
-    data: pd.DataFrame, use_mean: bool, ticker: str, adjusted_var: bool, percentile: int
+    data: pd.DataFrame,
+    use_mean: bool,
+    ticker: str,
+    adjusted_var: bool,
+    student_t,
+    percentile: int,
+    portfolio: bool,
```

Comment:
We should use as many default kwargs as we can here (and is es).  For example, if I just had a Dataframe of prices, I should be able to run `display_var( data)` instead of `display_var(data,True,True,....`. Also what if percentile is not provided?

Reply:
I added many default arguments. I won't reduce the number of parameters since they make the code way more usefull in a multiple of scenarios. The default percentile is set to 99.9%.